src/ui/public/partials/saved_object_finder.html

```diff
@@ -52,12 +52,19 @@
   class="kuiVerticalRhythm"
 >
   <ul class="li-striped list-group list-group-menu" ng-class="{'select-mode': finder.selector.enabled}">
-    <li class="list-group-item" ng-click="finder.sortHits(finder.hits)">
+    <li
+      class="list-group-item"
+      ng-click="finder.sortHits(finder.hits)"
+      kbn-accessible-click
+      aria-live="assertive"
+      >
       <span class="paginate-heading">
+        <span class="sr-only">Sort by</span>
```

Comment:
The UI Framework has a `kuiScreenReaderOnly` class. Can we use that instead of Bootstrap's `sr-only`?